spring-integration-sftp/src/test/java/org/springframework/integration/sftp/config/InboundChannelAdapterParserTests-context.xml

```diff
@@ -42,6 +42,7 @@
 								  temporary-file-suffix=".bar"
 								  comparator="comparator"
 								  local-filter="acceptAllFilter"
+								  max-fetch-size="42"
```

Comment:
Now let's look into the:
````xml
<inbound-channel-adapter id="jdbcAdapter" query="select * from item"
						max-fetch-size="42"	 
			data-source="dataSource" />
````
Does it sound good there?

I agree that existing `max-rows-per-poll` isn't good, too (https://jira.spring.io/browse/INT-2260), but will be the new name better? Maybe just `max-fetch`? Although `fetch` has a bit different meaning in JDBC...
OTOH our JPA Adapters provide `max-results` options...

However it doesn't matter how we call the target option if the meaning in the `MessageSource` is the same - `max-fetch-size`.

I can live with this, but rename/implement/remap those options in JDBC and JPA.
WDYT?